poglink/bot.py

```diff
@@ -87,9 +87,15 @@ def from_file(cls, config_path):
 
         return cls.from_dict(config_from_file)
 
+    def __repr__(self):
+        return f"BotConfig<rates_channel_id={self.rates_channel_id}, bans_channel_id={self.bans_channel_id}, polling_delay={self.polling_delay}>"
```

Comment:
Was annoying to troubleshoot when the interpreter only showed these objects as `ObjectName at <0xmemoryaddress`. This is just more readable when you want to print an instance without special formatting.